pre_commit/meta_hooks/check_files_matches_any.py

```diff
@@ -0,0 +1,36 @@
+import re
+import sys
+
+import pre_commit.constants as C
+from pre_commit.clientlib import load_config
+from pre_commit.git import get_all_files
+
+
+def files_matches_any(filenames, include):
```

Comment:
let's try and reuse the `include_exclude` from `pre_commit.commands.run` (such that this more reflects reality and doesn't drift)